src/test/java/by/itech/lab/supplier/CategoryTest.java

```diff
@@ -0,0 +1,36 @@
+package by.itech.lab.supplier;
+
+import by.itech.lab.supplier.domain.Category;
+import by.itech.lab.supplier.dto.CategoryDto;
+import by.itech.lab.supplier.service.impl.CategoryService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+public class CategoryTest {
+
+    @Autowired
+    CategoryService service;
+
+    @Test
+    void checkConnection() {
+        CategoryDto category = new CategoryDto();
+        category.setCategory("Test");
+        category.setTaxRate(20.0);
+        category.setActive(true);
+        category = service.save(category);
+        CategoryDto readCategory = service.findByCategory("Test");
+        assert category.equals(readCategory);
```

Comment:
can you please split logically your checks? basically, each test contains 3 parts:
- what we have - given
- what we do - when
- what we expect - then

for example we're testing method which sum 2 variables:
```
// given
final Integer a = 25;
final Integer b = 25

// when
final Integer result = someService.sum(a, b);

// then
assertThat(result).isEqualTo(a + b);
```

Reply:
done